src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-bar__big"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        name="search"
+        class="search-bar__big__field"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      action="#"
+      class="search-bar__small"
```

Comment:
```suggestion
      class="search-bar search-bar--small"
```
Это модификатор по БЭМу и должен еще быть общий класс, без модификатора, в котором будут общие стили для большого и маленького search-bar